lv_port/lv_port_display_espressif.c

```diff
@@ -66,48 +66,53 @@ bool display_port_gpio_is_busy(lv_disp_drv_t *drv)
     return device_busy;
 }
 
-void display_interface_send_cmd(lv_disp_drv_t *drv, uint8_t cmd, void *args, size_t args_len)
+void display_interface_send_cmd(lv_disp_drv_t *drv, uint32_t cmd, cmd_width_t cmd_width, void *args, size_t args_len)
```

Comment:
Just my 2 cents: I'd use a simple `uint8_t` instead of `cmd_width_t`.

Reply:
Done, handled as lvgl does